### PR TITLE
Blog post for Katy Huff's contributor questionnaire

### PR DESCRIPTION
**Pull request by amyrbrown**
This is the blog post with Katy Huff's answers to our contributor questionnaire. I didn't test it locally because Jekyll and Ruby are fighting back and it's time for me to make dinner -- sorry. :( 

Not sure if this should be posted immediately, or if there's a greater plan/timeline -- @adina?

### CODE REVIEW

File: blog/2015/03/better-know-katy-huff.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+root: ../../..
+author: Amy Brown
+title: "Better Know a SWC Contributor: Katy Huff"
```
Comment (gvwilson) [inline blog/2015/03/better-know-katy-huff.html]: I'd prefer "Getting to Know: Katy Huff"

File: blog/2015/03/better-know-katy-huff.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+date: 2015-03-15
+time: "19:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  This is the first in a series of posts about our contributors. We're posting
+  these so our community can get to know our community. If you'd like to be
+  profiled, or you'd like to nominate another member of the community, 
+  drop us a line.
```
Comment (gvwilson) [inline blog/2015/03/better-know-katy-huff.html]: Three uses of "community" in two sentences...

File: blog/2015/03/better-know-katy-huff.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+</p>
+<p>
+  Our first contributor profilee is Katy Huff. Katy is a member of our steering
```
Comment (gvwilson) [inline blog/2015/03/better-know-katy-huff.html]: Our first profile is of Katy Huff, a member of... who is also on...

**Comment by adina** [comment]
@amyrbrown There is no time line but I did want to get a manageable blog posts "banked" for some timed rolled out once we begin - I was thinking once - twice a month.  Also, @wking can probably help you figure out how to roll these out more efficiently - it was his instructions that helped me integrate with workshop websites - he's fantastic at this.

**Comment by amyrbrown** [comment]
Sorry. I'll get these mistakes fixed and push again. I didn't get the notifications for these -- better check my email settings.

File: blog/2015/03/better-know-katy-huff.html (hunk 1 above)
Comment (amyrbrown) [inline blog/2015/03/better-know-katy-huff.html]: Stupid Daily Show references not helpful -- got it.

File: blog/2015/03/better-know-katy-huff.html (hunk 2 above)
Comment (amyrbrown) [inline blog/2015/03/better-know-katy-huff.html]: This is why I'm an editor, not a writer.

**Comment by amyrbrown** [comment]
Fixes fixed.

File: blog/2015/03/getting-to-know-katy-huff.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+</p>
+
+<h3>If three phrases/words described you, what would they be?</h3>
+<p>
+Biking, vegetarian, nuclear engineer.
+</p>
+
+<h3>What research or work do you do?</h3>
+<p>
+I simulate various things involving nuclear energy. In my dissertation work, that was focused on the systems analysis of nuclear material movement between facilities in the nuclear fuel cycle. In my postdoc, I have been more focused on multiphysics analysis of neutronics and thermal hydraulics in a very special advanced reactor design, the Pebble-Bed, Fluoride-salt-cooled, High-temperature Reactor (PB-FHR).
```
Comment (wking) [inline blog/2015/03/getting-to-know-katy-huff.html]: Maybe “My dissertation work was focused on…”?

Comment (amyrbrown) [inline blog/2015/03/getting-to-know-katy-huff.html]: @wking, as the post "editor" I don't want to change the subjects' own wording unless it's to correct egregious errors. I guess it's up to @katyhuff to correct this if she wants to and then we can get it posted.

**Comment by amyrbrown** [comment]
@gvwilson Should I redo this PR with an updated date on the blog post?

**Comment by jiffyclub** [comment]
You might also link to Katy's blurb on our Team page: http://software-carpentry.org/pages/team.html#huff.k, or even pull in that blurb as a quote. (Unfortunately I don't know of a way to find the `huff.k` part of that except by inspecting the HTML.)

Having a little biographic information before diving into the questions is helpful, I think.

**Comment by amyrbrown** [comment]
Good idea, @jiffyclub . I slipped in the link at the first mention of Katy's name.

**Comment by katyhuff** [comment]
@amyrbrown @adina : To implement @wking's grammar suggestion, I made a PR to this PR here: https://github.com/amyrbrown/site/pull/1

Unless there are more grammar issues, this is fine to go out whenever, as far as I'm concerned.

File: blog/2015/03/getting-to-know-katy-huff.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+title: "Getting to Know: Katy Huff"
+date: 2015-03-15
+time: "19:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  This is the first in a series of posts about our contributors. We're posting
+  these so our community can get to know each other better. If you'd like to be
+  profiled, or you'd like to nominate another member, drop us a line.
```
Comment (r-gaia-cs) [inline blog/2015/03/getting-to-know-katy-huff.html]: @amyrbrown What about replace "drop us a line" with "send a email to communications@lists.software-carpentry.org"?

File: blog/2015/03/getting-to-know-katy-huff.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+</p>
+
+<h3>What research or work do you do?</h3>
+<p>
+I simulate various things involving nuclear energy. In my dissertation work, I focused on the systems analysis of nuclear material movement between facilities in the nuclear fuel cycle. In my postdoc, I have been more focused on multiphysics analysis of neutronics and thermal hydraulics in a very special advanced reactor design, the Pebble-Bed, Fluoride-salt-cooled, High-temperature Reactor (PB-FHR).
+</p>
+
+<h3>What programming languages and other technology do you regularly use?</h3>
+<ul>
+    <li>python (scipy, numpy, nose, pytables)</li>
```
Comment (r-gaia-cs) [inline blog/2015/03/getting-to-know-katy-huff.html]: Can you replace this list with

~~~
<li>Python (SciPy, NumPy, nose, PyTables)</li>
<li>C++</li>
<li>Git</li>
<li>CMake</li>
<li>Sphinx</li>
<li>doxygen</li>
<li>GoogleTest</li>
~~~

**Comment by gvwilson** [comment]
@amyrbrown @katyhuff is this one ready to merge?

File: blog/2015/04/getting-to-know-katy-huff.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+root: ../../..
+author: Amy Brown
+title: "Getting to Know: Katy Huff"
+date: 2015-03-15
+time: "19:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  This is the first in a series of posts about our contributors. We're posting
```
Comment (r-gaia-cs) [inline blog/2015/04/getting-to-know-katy-huff.html]: @gvwilson Can you fix this:

> This is the first in a series of posts about our contributors.

It should be

> This is the second in a series of posts about our contributors.

**Pull request merged**